client/src/elements/schema-editor/schema-editor-bbox.html

```diff
@@ -0,0 +1,10 @@
+<template>
+  <require from="./schema-editor-bbox.css"></require>
+  <div class="title-container">
+    <label class="input-group-title">
+      ${schema["title"] & toolT}
+    </label>
```

Comment:
I would add an explanation here like: "<Shift> gedrückt halten und Rechteck mit der Maus aufziehen" or something to explain how it is to be used.

Reply:
Implemented in https://github.com/nzzdev/Q-editor/pull/226/commits/cfeea81c494603d7eb347c1fda8e0fdc1abed466